cmd/clusterctl/cmd/describe_cluster_test.go

```diff
@@ -228,15 +230,15 @@ func Test_TreePrefix(t *testing.T) {
 			expectPrefix: []string{
 				"Object/root",
 				"├─Object/child1",
-				"│             ├─C1.1", // first condition child gets pipes and ├─
-				"│             └─C1.2", // last condition child gets └─ and pipes and └─
+				"│             ├─C1.1  True      0s", // first condition child gets pipes and ├─
+				"│             └─C1.2  True      0s", // last condition child gets └─ and pipes and └─
 				"└─Object/child2",
-				"              ├─C2.1", // first condition child gets spaces and ├─
-				"              └─C2.2", // last condition child gets spaces and └─
+				"              ├─C2.1  True      0s", // first condition child gets spaces and ├─
+				"              └─C2.2  True      0s", // last condition child gets spaces and └
```

Comment:
Do you still need these changes given the new matcher now uses `strings.Prefix`?